charts/discovery-ui/values.yaml

```diff
@@ -20,7 +20,10 @@ env:
   api_host: "https://data.example.com"
   #contribute_host: "https://sharedata.example.com" #Optional location for an associated public Andi deployment
   gtm_id: "GTM-EXAMPLE"
+  #used in react_discovery_ui for websocket command
```

Comment:
Similar to the other comment in this file about use case.

Might as well keep the specificity of what you added, could we also add:

```
# the host domain where urbanos is deployed
```